client/src/django.js

```diff
@@ -64,4 +76,17 @@ const djangoClient = new Vue({
   }
 });
 
+// This is done with an interceptor because the value of
+// oauthClient.authHeaders is initialized asynchronously,
+// and doesn't exist at all if the user isn't logged in.
+// Using client.defaults.headers.common.Authorization = ...
+// would not update when the headers do.
+apiClient.interceptors.request.use(config => ({
```

Comment:
I don't really know the exact "best" way to handle this, but [here's how I do it in my other Girder 4 SPAs.](https://github.com/girder/girder_web_components/blob/dkc-next/dkc/main.js#L27)

That seems a little more straightforward than putting this at the module level.